composer.json

```diff
@@ -26,6 +26,7 @@
         "symfony/framework-bundle": "~2.3"
     },
     "require-dev": {
+        "friendsofsymfony/http-cache": "~1.2@dev",
```

Comment:
Hm yes, but we don't want people to get the unstable version of the library when installing a tagged version of the bundle. Do we then need to keep the minimum-stability?